ElectronClient/gui/NoteText.jsx

```diff
@@ -1025,9 +1019,9 @@ class NoteTextComponent extends React.Component {
 				const range = this.getSelectionRange();
 				if (range.isEmpty()) {
 					const row = range.start.row;
-					const tokens = this.session.getTokens(row);
+					const tokens = listTokens(this, row);
 
-					if (tokens.length > 0 && tokens[0].type == 'markup.list') {
+					if (tokens) {
 						if (tokens[0].value.search(/\d+\./) != -1) {
```

Comment:
On seeing this, i think it would be clearer if `listTokens` always returns a list, and we go
```
if (tokens.length > 0)
```
No bug here, it just looks suspicious.